report/data_tasks/report/create_from_scratch/02_hubspot_read/01_tables/01_create_company_table.sql

```diff
@@ -1,10 +1,13 @@
+-- Once this has run once, it can be removed
 DROP TABLE IF EXISTS hubspot.companies CASCADE;
+DROP TABLE IF EXISTS hubspot.companies_raw CASCADE;
 
-CREATE TABLE hubspot.companies (
+CREATE TABLE hubspot.companies_raw (
    -- General
     id BIGINT PRIMARY KEY,
     name TEXT,
     lms_organization_id TEXT,
+    life_cycle_stage TEXT,
```

Comment:
It would be more efficient to use an enum here but...

 * We don't control these values and changes in Hubspot could break us any time
 * There aren't that many rows in this table that we need to be worried